test/onnx/gen_onnx.py

```diff
@@ -3231,6 +3232,89 @@ def min_test():
     return ([node], [a, b, c], [y])
 
 
+@onnx_test
+def mod_test():
+    a = helper.make_tensor_value_info('0', TensorProto.FLOAT, [3, 3, 3])
+    b = helper.make_tensor_value_info('1', TensorProto.FLOAT, [3, 3, 3])
+    y = helper.make_tensor_value_info('2', TensorProto.FLOAT, [3, 3, 3])
+
+    node = onnx.helper.make_node('Mod', inputs=['0', '1'], outputs=['2'])
```

Comment:
Shouldn't `fmod` be set to True always for `float` types ?

Reply:
No since it mirrors the functionality of pythons % or mod command.

Reply:
https://github.com/onnx/onnx/blob/main/docs/Operators.md#Mod 
"If the input type is floating point, then `fmod` attribute must be set to 1."